test/e2e/middleware-request-header-manipulation/app/next.config.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  i18n: {
+    locales: ['ja', 'en', 'fr', 'es'],
+    defaultLocale: 'en',
+  },
+  rewrites() {
+    return {
+      beforeFiles: [
+        {
+          source: '/beforefiles-rewrite',
+          destination: '/ab-test/a',
+        },
+      ],
+      afterFiles: [
+        {
+          source: '/afterfiles-rewrite',
+          destination: '/ab-test/b',
+        },
+        {
+          source: '/afterfiles-rewrite-ssg',
+          destination: '/fallback-true-blog/first',
+        },
+      ],
+      fallback: [],
+    }
+  },
+}
```

Comment:
**question:** why do we have a config here? this test does not use rewrites nor locale.

Reply:
oops this part is copied-and-pasted 🤦 Thanks for lmk!